src/components/ConvergenceMap.tsx

```
@@ -0,0 +1,350 @@
+  const mean = useMemo(() => getMean(values), [values]);
+
+  // Early return if no valid data
+  if (!Number.isFinite(median) || values.length === 0) {
+    return (
+      <div style={{ width, height, display: 'flex', alignItems: 'center', justifyContent: 'center' }}>
+        <div style={{ color: colors.ticks, fontSize: 14 }}>No estimates to display</div>
+      </div>
+    );
+  }
+
+  const medianIdx = useMemo(() => nearestScaleIndex(median), [median]);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)